packages/react/src/components/cookieConsent/page/Page.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+
+import { Content, Provider as CookieContextProvider } from '../CookieConsentContext';
+import Details from '../details/Details';
+import styles from '../CookieConsent.module.scss';
+import Buttons from '../buttons/Buttons';
+import { Notification } from '../../notification/index';
+
+export function Page(props: { content: Content; cookieDomain?: string }): React.ReactElement | null {
```

Comment:
Since this is the public component, I think we need to give it a name that has the word cookie in it. Maybe CookiePage?